src/kube-runtime/deploy/rbac.yaml

```diff
@@ -18,17 +18,28 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: frameworkbarrier-account
+  name: runtime-account
   namespace: default
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: frameworkbarrier-role
```

Comment:
why not change this name?

Reply:
It's the role name, means, runtime can act as such role. This RBAC yaml mean runtime service can act as two roles, one is frameworkbarrier, another is storage

---
In reply to: [352481764](https://github.com/microsoft/pai/pull/3944#discussion_r352481764) [](ancestors = 352481764)

Reply:
Maybe runtime-framework-role and runtime-storage-role are better names

---
In reply to: [353079012](https://github.com/microsoft/pai/pull/3944#discussion_r353079012) [](ancestors = 353079012,352481764)